lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
The previous sentence reads:

> x, y, xerr, and yerr can all be scalars, which plots a single error bar at x, y.

I think that should be enough for most users.